jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/SubscriptionResource.java

```diff
@@ -352,10 +351,12 @@ public Response createSubscriptionsWithAddOnsInternal(final List<BulkSubscriptio
 
             final LocalDate resolvedEntitlementDate = toLocalDate(entitlementDate);
             final LocalDate resolvedBillingDate = toLocalDate(billingDate);
+            final DateTime entitlementDateTime = resolvedEntitlementDate != null ? resolvedEntitlementDate.toDateTimeAtCurrentTime() : null; //TODO_1375 Revisit once JAX_RS changes are done
```

Comment:
As discussed - fix it using `TimeAwareContext`.

Reply:
Fixed